### PR TITLE
Fixed example and added method with argument

### PR DESCRIPTION
**Pull request by backtozero**
1) Fixed example code. It didn't worked.
2) My sensor lives on different i2c address than 0x77. So i added function that makes everything the same except it takes address argument. I guess there is more elegant way to do this, but i am new in Rust, so you can made it in better way.

I Don't expect you to accept this pull request. Just want to be helpful and make it more useful.

Thanks.

### CODE REVIEW

**Comment by backtozero** [comment]
There is same thing with i2cdev-lsm9ds0, but there is 2 addresses.
I guess it should be better to implement it as a builder pattern.

What do you think?

**Comment by backtozero** [comment]
I think it would be better to pass parameters in settings block as it already done with other options.
Closing this pull.

Will make another one which wouldn't break existence contract and will request it again.

File: i2cdev-bmp280/src/lib.rs [hunk 1]
```diff
@@ -167,6 +167,14 @@ pub fn get_linux_bmp280_i2c_device() -> Result<LinuxI2CDevice, LinuxI2CError> {
     }
 }
 
+#[cfg(any(target_os = "linux", target_os = "android"))]
+pub fn get_linux_bmp280_i2c_device_with_addr(addr: u16) -> Result<LinuxI2CDevice, LinuxI2CError> {
```
Comment (martindeegan) [inline i2cdev-bmp280/src/lib.rs:171]: This is a bit redundant. You could just call `LinuxI2CDevice::new("/dev/i2c-1", addr)` manually. Additionally, this doesn't have anything to do with the bmp280 anymore.

**Comment by martindeegan** [comment]
> There is same thing with i2cdev-lsm9ds0, but there is 2 addresses.
> I guess it should be better to implement it as a builder pattern.
> 
> What do you think?

Read my comment. I think this is redundant. The default builder is for someone using the address specified in the datasheet. Anyone who chose to change that address needs to adjust accordingly.

**Comment by martindeegan** [comment]
> I think it would be better to pass parameters in settings block as it already done with other options.
> Closing this pull.
> 
> Will make another one which wouldn't break existence contract and will request it again.

One thing that would be useful would be to implement the `Default` trait for the settings structs.

**Comment by backtozero** [comment]
Ok, i got you. Thanks.

**Pull request closed without merging**